TeamCode/src/main/java/org/firstinspires/ftc/teamcode/components/TeleOpMovement.kt

```diff
@@ -42,11 +46,18 @@ object TeleOpMovement : Component {
         val joyRadians = atan2(joyY, joyX) - (PI / 3.0)
         var joyMagnitude = sqrt(joyY * joyY + joyX * joyX)
 
+        update()
```

Comment:
Could you move the `update` call to the very top of the `loop` function body? I worry people will try accessing `GamepadEx` before `update` is called, which can cause issues.